mLRS/CommonTx/cli.h

```diff
@@ -639,36 +660,46 @@ char unit[32];
 
 void tTxCli::print_help(void)
 {
-    print_layout_version_warning();
-
-    putsn("  help, h, ?  -> this help page");
-    putsn("  v           -> print device and version");
-    putsn("  pl          -> list all parameters");
-    putsn("  pl c        -> list common parameters");
-    putsn("  pl tx       -> list Tx parameters");
-    putsn("  pl rx       -> list Rx parameters");
-
-    putsn("  p name          -> get parameter value");
-    putsn("  p name = value  -> set parameter value");
-    putsn("  p name = ?      -> get parameter value and list of allowed values");
-    putsn("  pstore      -> store parameters");
-
-    putsn("  setconfigid -> select config id");
-    putsn("  bind        -> start binding");
-    putsn("  reload      -> reload all parameter settings");
-    putsn("  stats       -> starts streaming statistics");
-    putsn("  listfreqs   -> lists frequencies used in fhss scheme");
+    if (output_index == 0) {
+        print_layout_version_warning();
+
+        putsn("  help, h, ?  -> this help page");
+        putsn("  v           -> print device and version");
+        putsn("  pl          -> list all parameters");
+        putsn("  pl c        -> list common parameters");
+        putsn("  pl tx       -> list Tx parameters");
+        putsn("  pl rx       -> list Rx parameters");
+
+        putsn("  p name          -> get parameter value");
+        putsn("  p name = value  -> set parameter value");
+        putsn("  p name = ?      -> get parameter value and list of allowed values");
+        putsn("  pstore      -> store parameters");
+        output_index++; // Do next chunk next time
```

Comment:
this here would be a clear example where one could use just the enum with more subsructure to step through the pieces